exercises/calc/calc.py

```diff
@@ -63,35 +70,35 @@ def parse(s, i, ts):
 
 def main():
 
-    p = make_seq(num_parser, make_seq(op_parser,num_parser))
+    p = make_seq(num_parser, make_seq(op_parser,num_parser))  # Create parser for number and operator sequence
     s = ''
     iface = 'eth0'
 
     while True:
-        s = input('> ')
+        s = input('> ')  
```

Comment:
This is a nitpick, but in general adding whitespace at the end of lines would be nice to avoid, especially if there is nothing else being changed.

Reply:
> This is a nitpick, but in general adding whitespace at the end of lines would be nice to avoid, especially if there is nothing else being changed.

Every advise is helpful and counts. I'll surely keep that in mind